### ISSUE TITLE
(Apparently) cannot serve assets from root

### ISSUE THREAD

**Issue by flavor8** [bazaarvoice/dropwizard-configurable-assets-bundle#28]
Dropwizard core allows the application (API) to be served from a sub-URL, and assets from the root. This is done via:

public void initialize(Bootstrap<HelloWorldConfiguration> bootstrap) {
    bootstrap.addBundle(new AssetsBundle("/assets/", "/"));
}

There doesn't appear to be a similar structure for ConfiguredAssetsBundle. If I try to set the resourcePath to "/", it throws an exception:

Exception in thread "main" java.lang.IllegalArgumentException: / is the classpath root
    at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
    at com.bazaarvoice.dropwizard.assets.ConfiguredAssetsBundle.<init>(ConfiguredAssetsBundle.java:106)
    at com.bazaarvoice.dropwizard.assets.ConfiguredAssetsBundle.<init>(ConfiguredAssetsBundle.java:63)
    at com.meddata.pop.PopulationBuilderApplication.initialize(PopulationBuilderApplication.java:24)
    at io.dropwizard.Application.run(Application.java:70)
    at com.meddata.pop.PopulationBuilderApplication.main(PopulationBuilderApplication.java:37)
# 

EDIT:
I may be confused by the documentation. You have "dashboard" as both the uripath and the resource name in the yaml overrides section. What happens to the "/assets/" resource path in the case of externally served resources?

Bottom line, I want to serve the url "/" using static assets in "/usr/local/myproject/assets". What's the way to configure that?

Thanks!
